### PR TITLE
added names on sign-up form and validation

### PR DESCRIPTION
**Pull request by sophiecs333**
I think it works fine, can you please check?

### CODE REVIEW

File: app/views/devise/registrations/new.html.erb [hunk 1]
```diff
@@ -10,6 +10,14 @@
             <%= f.error_notification %>
 
             <div class="form-inputs">
+              <%= f.input :first_name,
+                          required: true,
+                          autofocus: true,
+                          input_html: { autocomplete: "first_name" }%>
+              <%= f.input :last_name,
+                          required: true,
+                          autofocus: true,
```
Comment (dumboluzz) [inline app/views/devise/registrations/new.html.erb:19]: autofocus only makes sense on 1 form input. Autofocus means this input field will be focused when the page loads.

**Review by finnstue — APPROVED** [review]
looks good!

**Review by dumboluzz — DISMISSED** [review]
just a small mistake doesnt matter really

**Review by finnstue — APPROVED** [review]
looks good

**Comment by dumboluzz** [comment]
@sophiecs333 Sign up doesn't work because first_name and last_name are not recognized.

**Pull request merged**